FateGrandAutomata/ScriptRunnerService.cs

```diff
@@ -167,6 +167,21 @@ void StopScript()
             ShowStatusNotification("Ready");
         }
 
+
+        public override void OnTaskRemoved(Intent rootIntent)
+        {
+            // from https://stackoverflow.com/a/43310945/5971497
+
+            Intent restartServiceIntent = new Intent(this.ApplicationContext, this.Class);
+            restartServiceIntent.SetPackage(this.PackageName);
+
+            PendingIntent restartServicePendingIntent = PendingIntent.GetService(this.ApplicationContext, 1, restartServiceIntent, PendingIntentFlags.OneShot);
+            AlarmManager alarmService = (AlarmManager)ApplicationContext.GetSystemService(Context.AlarmService);
+            alarmService.Set(AlarmType.ElapsedRealtime, SystemClock.ElapsedRealtime() + 1000, restartServicePendingIntent);
+
+            base.OnTaskRemoved(rootIntent);
+        }
```

Comment:
```suggestion
        public override void OnTaskRemoved(Intent RootIntent)
        {
            // from https://stackoverflow.com/a/43310945/5971497

            var restartServiceIntent = new Intent(ApplicationContext, Class);
            restartServiceIntent.SetPackage(PackageName);

            var restartServicePendingIntent = PendingIntent.GetService(ApplicationContext, 1, restartServiceIntent, PendingIntentFlags.OneShot);
            var alarmService = (AlarmManager)ApplicationContext.GetSystemService(AlarmService);
            alarmService.Set(AlarmType.ElapsedRealtime, SystemClock.ElapsedRealtime() + 1000, restartServicePendingIntent);

            base.OnTaskRemoved(RootIntent);
        }
```

Just a few conventions I follow.
I'm just trying whether I can commit to your branch since you asked.